tests/sentry/api/endpoints/test_organization_events_anomalies.py

```diff
@@ -0,0 +1,294 @@
+from datetime import timedelta
+from unittest.mock import patch
+
+import orjson
+from urllib3 import HTTPResponse
+from urllib3.exceptions import TimeoutError
+
+from sentry.incidents.models.alert_rule import (
+    AlertRuleSeasonality,
+    AlertRuleSensitivity,
+    AlertRuleThresholdType,
+)
+from sentry.seer.anomaly_detection.types import (
+    Anomaly,
+    AnomalyDetectionConfig,
+    DetectAnomaliesResponse,
+    DetectHistoricalAnomaliesContext,
+    TimeSeriesPoint,
+)
+from sentry.seer.anomaly_detection.utils import translate_direction
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.helpers.datetime import before_now, freeze_time, timestamp_format
+from sentry.testutils.helpers.features import with_feature
+from sentry.testutils.outbox import outbox_runner
+
+
+@freeze_time()
+class OrganizationEventsAnomaliesEndpointTest(APITestCase):
+    endpoint = "sentry-api-0-organization-events-anomalies"
+
+    method = "post"
+
+    four_weeks_ago = before_now(days=28).replace(hour=10, minute=0, second=0, microsecond=0)
+    one_week_ago = before_now(days=7)
+
+    config = AnomalyDetectionConfig(
+        time_period=60,
+        sensitivity=AlertRuleSensitivity.LOW.value,
+        direction=translate_direction(AlertRuleThresholdType.ABOVE.value),
+        expected_seasonality=AlertRuleSeasonality.AUTO.value,
+    )
+    historical_timestamp_1 = timestamp_format(four_weeks_ago)
+    historical_timestamp_2 = timestamp_format(four_weeks_ago + timedelta(days=10))
+    current_timestamp_1 = timestamp_format(one_week_ago)
+    current_timestamp_2 = timestamp_format(one_week_ago + timedelta(minutes=10))
```

Comment:
nice! I like how easy this is to reason about -- ts1 is first ts2 is always second and freezing time 💯.